### DISCUSSION TITLE
Suçlu kim sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#204] [Zeka Sorusu]
Sitemizde işlenen bir suçun zanlılarını sorguya çekiyoruz. Suçlunun/suçluların yalan söylediği, masum/masumların ise doğru söylediğini biliyoruz. Zanlıların ifadeleri şu şekildedir.
Duygu: Filiz suçludur.
Filiz : Duygu ile Melike'nin ikisi de masumdur.
Melike: Duygu ile Filizin ya ikisi birden suçludur ya da ikisi birden masumdur.
Suçluyu/suçluları bulunuz.

**Comment by Alperg123456** [comment]
Duygunun masum olduğunu ve doğruyu söylediğini varsayalım. Duygu, Filiz'in masum olduğunu söylediğinden ve bunun doğru olduğunu varsaydığımız için, 
bu durumda Filiz'nun önermesi de doğru olur. Filiz de Duygu ve Melike'nin masum olduğunu önerir, bunu doğru kabul ediyoruz, ilk varsayımımızı devam ettirerek.
Bu senaryoda Melike'nin önermesi de doğru olur, Duygu ve Filiz de aynı anda masumdur. Sonuç olarak, Duygu'nun söylediklerini doğru kabul edersek herkes masumdur. 
İlk hipotez mantıklı bir sonuç vermedi, eğer ortada bir suç varsa ve suçu işleyebilecek insanlar Duygu Filiz veya Melike'nin arasında ise farklı bir varsayımla 
ilerlemek gerekir. 
Duygunun suçlu olduğunu ve yalan söylediğini varsayalım. "Filiz masumdur" bir yalan olduğuna göre Filiz de suçludur ve önermesi de yanlıştır. "Duygu ve Melike suçsuzdur."
dikkate alınmamalıdır. Duygu veya Melike suçlu veya masum olabilir, bilemeyiz. Melike de varsayımımızın söylediğini destekler nitelikte doğru söylemektedir. Duygu ve Filiz
aynı anda suçludur, bu Melike'nin önermesini doğru kılar ve dolayısıyla masumdur. İki adet çözüm önerisine ulaşabiliriz:

1.Hipotez (Duygu Masumdur ve doğruyu söyler)

Duygu - Masum
Filiz - Masum
Melike - Masum

2.Hipotez (Duygu Suçludur ve yalan söyler)

Duygu * Suçlu
Filiz * Suçlu
Melike - Masum
